// sendchamp-sdk/lib/services/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.CALL = exports.EMAIL = exports.VOICE = exports.VERIFICATION = exports.WHATSAPP = exports.SMS = void 0;
var sms_1 = require("./sms");
Object.defineProperty(exports, "SMS", { enumerable: true, get: function () { return __importDefault(sms_1).default; } });
var whatsapp_1 = require("./whatsapp");
Object.defineProperty(exports, "WHATSAPP", { enumerable: true, get: function () { return __importDefault(whatsapp_1).default; } });
var verification_1 = require("./verification");
Object.defineProperty(exports, "VERIFICATION", { enumerable: true, get: function () { return __importDefault(verification_1).default; } });
var voice_1 = require("./voice");
Object.defineProperty(exports, "VOICE", { enumerable: true, get: function () { return __importDefault(voice_1).default; } });
var email_1 = require("./email");
Object.defineProperty(exports, "EMAIL", { enumerable: true, get: function () { return __importDefault(email_1).default; } });
var call_1 = require("./call");
Object.defineProperty(exports, "CALL", { enumerable: true, get: function () { return __importDefault(call_1).default; } });
