// sendchamp-sdk/lib/constants/endpoints.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.baseUrl = void 0;
var baseUrl = Object.freeze({
    LIVE: "https://api.sendchamp.com/api/v1/",
    TEST: "https://sandbox-api.sendchamp.com/api/v1/",
    "LOCAL-SIMULATOR": "http://localhost:2920/api/v1/",
});
exports.baseUrl = baseUrl;
var endpoints = Object.freeze({
    NUMBER_INSIGHT: "number-insights/check",
    WALLET_BALANCE: "wallet/wallet_balance",
    SEND_SMS: "sms/send",
    REGISTER_SENDER: "sms/create-sender-id",
    SEND_VOICE: "voice/send",
    SEND_VERIFICATION_OTP: "verification/create",
    VERIFY_VERIFICATION_OTP: "verification/confirm",
    // NOTE: Tests not yet confirmed
    SEND_WHATSAPP: "whatsapp/message/send",
    VERIFY_WHATSAPP: "/whatsapp/validate",
    getCallReport: function (call_id) { return "number/call/".concat(call_id); },
    CREATE_VOICE_CALL: "number/call/create",
    LIST_VOICE_CALLS: "number/call/list",
    SEND_EMAIL: "email/send",
});
exports.default = endpoints;
