// sendchamp-sdk/lib/constants/types.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SMSRoute = exports.VoiceType = exports.SenderUseCase = exports.WhatsAppType = exports.InsightType = exports.TokenType = exports.Channel = exports.SendchampMode = exports.SendchampStatus = void 0;
var SendchampStatus;
(function (SendchampStatus) {
    SendchampStatus["success"] = "success";
    SendchampStatus["error"] = "error";
})(SendchampStatus || (exports.SendchampStatus = SendchampStatus = {}));
var SendchampMode;
(function (SendchampMode) {
    SendchampMode["test"] = "test";
    SendchampMode["live"] = "live";
    SendchampMode["local-simulator"] = "local-simulator";
})(SendchampMode || (exports.SendchampMode = SendchampMode = {}));
var Channel;
(function (Channel) {
    Channel["sms"] = "sms";
    Channel["email"] = "email";
    Channel["whatsapp"] = "whatsapp";
    Channel["voice"] = "voice";
})(Channel || (exports.Channel = Channel = {}));
var TokenType;
(function (TokenType) {
    TokenType["numeric"] = "numeric";
    TokenType["alphanumeric"] = "alphanumeric";
})(TokenType || (exports.TokenType = TokenType = {}));
var InsightType;
(function (InsightType) {
    InsightType["basic"] = "basic";
    InsightType["core"] = "core";
    InsightType["standard"] = "standard";
})(InsightType || (exports.InsightType = InsightType = {}));
var WhatsAppType;
(function (WhatsAppType) {
    WhatsAppType["template"] = "template";
    WhatsAppType["video"] = "video";
    WhatsAppType["audio"] = "audio";
    WhatsAppType["location"] = "location";
    WhatsAppType["sticker"] = "sticker";
    WhatsAppType["text"] = "text";
})(WhatsAppType || (exports.WhatsAppType = WhatsAppType = {}));
var SenderUseCase;
(function (SenderUseCase) {
    SenderUseCase["transactional"] = "Transactional";
    SenderUseCase["marketting"] = "Marketing";
    SenderUseCase["transactional & marketing"] = "Transactional & Marketing";
})(SenderUseCase || (exports.SenderUseCase = SenderUseCase = {}));
var VoiceType;
(function (VoiceType) {
    VoiceType["outgoing"] = "outgoing";
    VoiceType["incoming"] = "incomming";
    VoiceType["template"] = "template";
})(VoiceType || (exports.VoiceType = VoiceType = {}));
var SMSRoute;
(function (SMSRoute) {
    SMSRoute["non_dnd"] = "non_dnd";
    SMSRoute["dnd"] = "dnd";
    SMSRoute["international"] = "international";
})(SMSRoute || (exports.SMSRoute = SMSRoute = {}));
